### DISCUSSION TITLE
Next 13 metadata not readable by browser search engines for SEO

### DISCUSSION THREAD

**Discussion by JohnVersus** [vercel/next.js#47308] [Help]
### Summary

I dont know how correct I am on this but I recently made a webpage using next 13 and it has metadata generated inside layout.js. 

If I test my page SEO using google lighthouse it gives me a top SEO score. But when I check my page on search results it does not generate any SEO metadata information. 

Did anyone observe this in the next js apps? or Am I doing something wrong? 

I can share my page URL here if someone wants to verify. 

Thanks

**Comment by icyJoseph** [comment]
Hi,

Your site is crashing during SSR, and then CSR hydrates it successfully. That's why you can't find your metadata, but the site "works".

I can't pin the reason why it crashes during SSR though, at least not yet.

**Reply by JohnVersus** [reply]
How I can test if my site crashes during SSR?

I am not 100% sure if that is the reason. I do see some server-rendered components returned during the initial load.
